american-journal-of-climate-change.csl

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>American Journal of Climate Change</title>
+    <title-short>AJCC</title-short>
+    <id>http://www.zotero.org/styles/american-journal-of-climate-change</id>
+	<link href="http://www.zotero.org/styles/american-journal-of-climate-change" rel="self"/>
+	<issn>2167-9495</issn>
+      <eissn>2167-9509</eissn>
+	<link href="http://www.scirp.org/journal/ajcc/" rel="documentation"/>
+      <contributor>
+      <name>Arunkumar R</name>
+      <uri>http://www.mendeley.com/profiles/arunkumar-r1/</uri>
+    </contributor>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
+    <summary>A style for American Journal of Climate Change that includes article titles in the reference list</summary>
+    <updated>2015-12-16T21:10:23+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author" suffix=" ">
+      <name and="text" initialize-with="." name-as-sort-order="all"/>
+      <label form="short" prefix=", "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+      </substitute>
+    </names>
+    <text macro="year-date" prefix="(" suffix=") "/>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name initialize-with="." delimiter=", " delimiter-precedes-last="always"/>
+      <label form="short" prefix=" (" text-case="capitalize-first" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <text variable="publisher" suffix=", "/>
+    <text variable="publisher-place" suffix=".  "/>
```

Comment:
I actually think this period needs to go. Book chapter end with "place, 12-14", which this prevents